upstream/solr6/local/92-init-solr.sh

```diff
@@ -109,6 +109,17 @@ function makeConfigs {
         setOption 'alfresco.suggestable.property.2' '{http://www.alfresco.org/model/content/1.0}description' "$SHARED_PROPERTIES"
         setOption 'alfresco.suggestable.property.3' '{http://www.alfresco.org/model/content/1.0}content' "$SHARED_PROPERTIES"
     fi
+
+    # Load envvars starting with SHARED_ and put them in shared.properties
+    for i in $(env)
+    do
+      if [[ "$i" = SHARED_* ]]
+      then
+            key="$(echo "$i" | cut -d '=' -f 1 | cut -d '_' -f 2-)"
```

Comment:
`key=echo $i | cut -d '=' -f 1 | cut -d '_' -f 3-`  is the assignment of key in the code above. Can you double check?

Reply:
Shellcheck flags that usage with a couple of warnings, suggesting this instead.

```
In upstream/solr6/local/92-init-solr.sh line 67:
            key=`echo $i | cut -d '=' -f 1 | cut -d '_' -f 3-`
                ^-- SC2006: Use $(..) instead of legacy `..`.
                      ^-- SC2086: Double quote to prevent globbing and word splitting.
```

Reply:
I was referring to the  **cut -d '_' -f 2-** and **cut -d '_' -f 3-`** difference. Should have made that more clear.

Reply:
see line https://github.com/xenit-eu/docker-solr/blob/d3f827da91d453948d05ede9891e2622f3006bb2/upstream/solr6/local/92-init-solr.sh#L67

Reply:
Ah okay. The reason for that difference is that above, we are going from `GLOBAL_x_y` to `y`.
Here, we are going from `SHARED_x` to `x`.

`-f` selects which "fields" (things separated by the delimiter (`-d`)) are included.
For the first case, we need fields 3 and further.

For this `SHARED_` case, we need fields 2 and further.